routes/web.php

```diff
@@ -31,7 +31,8 @@
 });
 
 Route::middleware(['auth:web', \App\Http\Middleware\AccessTokenValidationMiddleware::class, \App\Http\Middleware\ForceOverseatingRedirectMiddleware::class, \App\Http\Middleware\ConventionDateRestrictionMiddleware::class])->group(function () {
-    Route::get('join', [\App\Http\Controllers\Applications\InvitationController::class, 'view'])->name('join');
+    Route::post('invitation/join', [\App\Http\Controllers\Applications\InvitationController::class, 'view'])->name('invitation.join');
```

Comment:
Calling GET on both /invitation/confirm and /invitation/join leads to a MethodNotAllowedHttpException.
We have redirects to the dashboard for similar cases, these two should be handled the same.